fakeredis/_server.py

```diff
@@ -2802,9 +2804,6 @@ def from_url(cls, url, db=None, **kwargs):
         pool = self.connection_pool
         pool.connection_class = FakeConnection
         pool.connection_kwargs['server'] = server
-        for key in ['password', 'host', 'port', 'path']:
-            if key in pool.connection_kwargs:
-                del pool.connection_kwargs[key]
```

Comment:
This wasn't really related, but it's no longer necessary now that FakeConnection can absorb arbitrary kwargs.